Makefile

```diff
@@ -142,3 +143,19 @@ clean:
 	rm -rf .cache
 	rm -rf $(BIN)
 	rm -rf $(RELEASE)
+
+# Package for Debian
+debpackage: DEBVERSION ?= $(shell echo $(VERSION) | sed "s/^v//" | \
```

Comment:
Don't we want to just use `lsb_release -sr` and move variables to the top of Makefile?